docs-devsite/firestore_.firestoredataconverter.md

```diff
@@ -10,89 +10,91 @@ https://github.com/firebase/firebase-js-sdk
 {% endcomment %}
 
 # FirestoreDataConverter interface
```

Comment:
I'm wondering if this is a good place to add general documentation about concepts of app model and DB model. Thoughts?

Specifically, target a user that may be new to the concepts. Maybe include a code example demonstrating how these concepts, and using a data converter is useful?

Reply:
Sure, I'll update the docs here.

Reply:
Done in https://github.com/firebase/firebase-js-sdk/pull/7932